bazel/repository_locations.bzl

```diff
@@ -1,9 +1,9 @@
 REPOSITORY_LOCATIONS = dict(
     envoy = dict(
-        # envoy 1.26.4 forked with extproc changes
+        # envoy 1.26.5 forked with extproc changes
         # sourced from release/v1.26-backportedfork
```

Comment:
Should we merge these changes before bumping this dependency? I feel like it could be tricky to make additional adjustments to the fork if we have to remember to pull in this branch first